contracts/PositionManager.sol

```diff
@@ -88,10 +88,6 @@ contract PositionManager is IPositionManager, ERC721Holder {
     ///@notice remove awareness of UniswapV3 NFT at index
     ///@param index index of the NFT in the uniswapNFTs array
     function removePositionIdAtIndex(uint256 index) internal {
```

Comment:
Why did we remove the require?

Reply:
Since the function is internal, that require is obviously satisfied, because the caller will always be this contract